### PR TITLE
[de] Add area slots for scenes

### PR DESCRIPTION
**Pull request by andreasbrett**
allow activation of area-specific scenes by adding area slots (see https://github.com/home-assistant/intents/issues/1788)

### CODE REVIEW

File: tests/de/scene_HassTurnOn.yaml [hunk 1]
```diff
@@ -25,3 +25,29 @@ tests:
...
+      - "führe die Partymodus Szene im Schlafzimmer aus"
+      - "aktiviere den Partymodus im Schlafzimmer"
+      - "aktiviere die Szene Partymodus im Schlafzimmer"
+      - "aktiviere die Partymodus Szene im Schlafzimmer"
+      - "Partymodus im Schlafzimmer"
+    intent:
+      name: HassTurnOn
+      slots:
+        area: Schlafzimmer
+		domain: scene
```
Comment (mib1185) [inline tests/de/scene_HassTurnOn.yaml:51]:
```suggestion
        domain: scene
```

File: sentences/de/scene_HassTurnOn.yaml [hunk 2]
```diff
@@ -14,3 +14,13 @@ intents:
...
+      - sentences:
+          - "[<szene>] <name> <area> [<aktivieren>|<ausfuehren>|<an>]"
+          - "<name> <szene> <area> (<aktivieren>|<ausfuehren>|<an>)"
+          - "führe [<szene>] <name> <area> aus"
+          - "führe <name> <szene> <area> aus"
+          - "<aktivieren> [<szene>] <name> <area>"
+          - "<aktivieren> <name> <szene> <area>"
+        slots:
+          domain: scene
+        response: scene
```
Comment (mib1185) [inline sentences/de/scene_HassTurnOn.yaml:26]: there are missing blank lines at the end of the files

**Review by mib1185 — CHANGES_REQUESTED** [review]
Hi @andreasbrett 

thanks for this contribution 👍 
There are only some small comments from my side.

**Comment by andreasbrett** [comment]
Hi @mib1185,
thanks for the comments, fixed both issues.

**Comment by mib1185** [comment]
Hi @andreasbrett 
I guess this PR should be **Ready for review** now (_short hint about https://github.com/home-assistant/intents/pull/1830#issuecomment-1870395280_)?

**Review by mib1185 — APPROVED** [review]

**Pull request merged**